timon/tst_helpers/sched_sim.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+import csv
+import os
+import random
+
+import yaml
+
+MYDIR = os.path.dirname(__file__)
+
+CFG_TEMPLATE_FNAME = os.path.join(MYDIR, "cfg.yaml")
+
+EXP_FNAME = "expected_seq.csv"
+
+sched_big = (
+    [(60, 30)] * 20
+    + [(120, 59)] * 20
+    + [(240, 5)] * 20
+    )
+
+sched_small = (
+    [(2, 1)] * 1
+    + [(4, 2)] * 1
+    + [(3, 1)] * 1
+    )
+
+sched_tiny = (
+    [(2, 1)] * 1
+    )
+
+
+class Probe:
+    def __init__(self, idx, entry):
+        self.name = f"{idx:03d}_{entry[0]:03d}_{entry[1]:03d}"
+        self.dly_ok = entry[0]
+        self.dly_ko = entry[1]
+        self.sequence = [bool(random.randint(0, 1)) for v in range(100)]
+        self.times = times = []
+        t = 0
+        for isok in self.sequence:
+            times.append(t)
+            t += entry[0] if isok else entry[1]
+
+
+def mk_probes(sched):
+    rslt = []
+    for idx, entry in enumerate(sched):
+        # print(f"{idx}: {entry}")
```

Comment:
do you want to keep this print? (can close)

Reply:
not really. the code seems to wrok and it easy to add. Thanks

Reply:
fixed in c7d0868a28693290b996956b06d9cb2a1688eaa2